.azure/azure-pipelines.ci.yml

```diff
@@ -203,6 +203,23 @@ stages:
 # Performance Tests
 #
 
+- stage: remoteperformance
+  displayName: Remote Performance Testing
+  dependsOn:
+  - build_windows
+  jobs:
+  - template: ./templates/run-remote-performance-int.yml
+    parameters:
+      localTls: schannel
+      remoteTls: schannel
+      extraArgs: '-Publish -Record'
+  - template: ./templates/run-remote-performance-int.yml
+    parameters:
+      localTls: schannel
+      remoteTls: schannel
+      extraName: '_Loopback'
```

Comment:
Can we pass this without the `_` and then add the `_` as necessary in the template?